### DISCUSSION TITLE
A general update

### DISCUSSION THREAD

**Discussion by asim** [micro/go-micro#2090] [General]
After much chaos and time reflecting on my efforts over the past 6 years I've realised Go Micro is end of life. I have attempted to carry that forward in Micro itself by consolidating interfaces there and I've looked at evolving Go Micro into something called Nitro for embeddable distributed app development as a way of removing confusing between Go Micro and Micro but in all that effort and experimentation I have realised it's just not possible. Go Micro now needs to left as is. I will no longer continue its development, but I also don't want to destroy what I spent many years building and what offered a lot of people value.

Go Micro will remain here, I make that promise to the community. For those that want to continue to use it, please do so and I apologise for the chaotic few weeks. For those who want to maintain it. I am open to conversations about handing over the project to those that might feel they can offer a better home for it. Go Micro effectively gave me the ability to raise funding for my company but became the victim of complexity in an effort to build a platform. In a way I think it lost its simplicity and elegance.

**Comment by crazybber** [comment]
you are welcome，we all need some experience to feel.

**Comment by crazybber** [comment]
I think ，maybe，it is a good manner to leave go-micro and its plugins as they are ，only receive PR for public maintaining ， you can focus your eyes more on micro.

**Comment by asim** [comment]
In light of this I'm doing some cleanup but what I welcome is some PRs for anything broken in go-micro, go-plugins, examples, etc
